smokeping.json

```diff
@@ -0,0 +1,48 @@
+{
+	"SmokePing": {
+		"website": "https://hub.docker.com/r/linuxserver/smokeping/",
+		"version": "1.0",
+		"description": "SmokePing is a network latency history monitor.",
+		"ui": {
+			"slug": "smokeping/smokeping.cgi"
+		},
+		"containers": {
+			"SmokePing": {
+				"image": "linuxserver/smokeping",
+				"launch_order": 1,
+				"volume_add_support": true,
+				"ports": {
+					"80": {
+						"description": "SmokePing WebUI port. Suggested default: 7878",
+						"host_default": 80,
+						"label": "WebUI port",
+						"protocol": "tcp"
+					}
```

Comment:
The ui_slug is correct, but you need to set this port as the webUI port in order for Rockstor to create the appropriate "webUI" button.
The `"ports"` section simply needs an extra line with `"ui": true`, as follows:
```
"protocol": "tcp",
"ui": true
```